### PR TITLE
Tisnew 5043 create concern

### PR DESCRIPTION
**Pull request by ojemuyiwa**

### CODE REVIEW

File: src/app/concern/concern.interfaces.ts [hunk 1]
```diff
@@ -10,6 +10,11 @@ export interface IListFile {
   key: string;
 }
 
+export interface IFileUploadeProgress {
```
Comment (omarCreativeDev) [inline src/app/concern/concern.interfaces.ts:13]: is it possible to fix the spelling of the interface name?

File: src/app/concern/file-uploader/file-uploader.component.ts [hunk 2]
```diff
@@ -24,27 +25,36 @@ export class FileUploaderComponent implements OnInit {
     "text/plain",
     "text/csv"
   ];
+  public acceptedFileSize = 1048576;
```
Comment (omarCreativeDev) [inline src/app/concern/file-uploader/file-uploader.component.ts:28]: i think you should speak to ify about a max file size limit as the requirement in the story is to not have a file size restriction...

File: src/app/concern/concern.interfaces.ts (hunk 1 above)
Comment (ojemuyiwa) [reply src/app/concern/concern.interfaces.ts]: Good catch: fixed

File: src/app/concern/file-uploader/file-uploader.component.ts (hunk 2 above)
Comment (ojemuyiwa) [reply src/app/concern/file-uploader/file-uploader.component.ts]: Absolutely after chatting on the group: I think Ify is fine with the solution, as she re-worded the text during conversations to 10MB

Comment (omarCreativeDev) [reply src/app/concern/file-uploader/file-uploader.component.ts]: sounds good

**Pull request merged**